src/components/dropdown/dropdown.svelte

```diff
@@ -48,11 +48,12 @@
 </script>
 
 <div class="leo-dropdown">
-  <div bind:this={dropdown}>
+  <div>
     <FormItem
       bind:disabled
       bind:required
       bind:size
+      bind:controlElement={dropdown}
```

Comment:
Previously, the popup would expand to the width of the control _and_ the label, if the label was horizontally aligned. This ensures that the popup width is based on the control, and not the label.